test/cookbooks/mongodb-test/templates/default/sv-mongodb-run.erb

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+# This service template originates from the test cookbook
+exec 2>&1
```

Comment:
Ah- that’s in the test cookbook.  Weird.  I made changes to the template itself.  I will follow up with the changes in test.

From: Damian Pantano <notifications@github.com<mailto:notifications@github.com>>
Reply-To: needle-cookbooks/chef-mongodb-composable <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, February 12, 2015 at 2:01 PM
To: needle-cookbooks/chef-mongodb-composable <chef-mongodb-composable@noreply.github.com<mailto:chef-mongodb-composable@noreply.github.com>>
Cc: Matt Jones <matt.jones@needle.com<mailto:matt.jones@needle.com>>
Subject: Re: [chef-mongodb-composable] Lots of Cameron and a little Jones (#6)


In test/cookbooks/mongodb-test/templates/default/sv-mongodb-run.erb<https://github.com/needle-cookbooks/chef-mongodb-composable/pull/6#discussion_r24619800>:

> @@ -0,0 +1,4 @@
> +#!/bin/sh
> +# This service template originates from the test cookbook
> +exec 2>&1


This is missing the ulimit changes you made.

—
Reply to this email directly or view it on GitHub<https://github.com/needle-cookbooks/chef-mongodb-composable/pull/6/files#r24619800>.